textile/features.textile

```diff
@@ -1618,18 +1627,41 @@ h4. ObjectData
 * @(OD1)@ An @ObjectData@ represents a value in an object on a channel
 * @(OD2)@ The attributes available in an @ObjectData@ are:
 ** @(OD2a)@ @objectId@ string - a reference to another object
-** @(OD2b)@ @encoding@ string - can be set by the client to indicate that value in @string@ or @bytes@ field have an encoding
-** @(OD2c)@ @boolean@ boolean - a primitive boolean leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - can be set at a time
-** @(OD2d)@ @bytes@ binary - a primitive binary leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - can be set at a time
-** @(OD2e)@ @number@ number - a primitive number leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - can be set at a time
-** @(OD2f)@ @string@ string - a primitive string leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - can be set at a time
+** @(OD2b)@ @encoding@ string - may be set by the client library to indicate that value in @string@ field have an encoding
+** @(OD2c)@ @boolean@ boolean - a primitive boolean leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - must be set at a time
+** @(OD2d)@ @bytes@ binary - a primitive binary leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - must be set at a time
```

Comment:
```suggestion
** @(OD2d)@ @bytes@ binary | base64 string - a primitive binary leaf value in the object graph. Only one of the value fields - @boolean@, @bytes@, @number@ or @string@ - must be set at a time
```
In case of `Json` encoding, we set `binary` field as base64 encoded `string`.